chimera/R3BAsyChimeraContFact.cxx

```diff
@@ -0,0 +1,83 @@
+// ----------------------------------------------------------------------
+//								    -----
+//  Factory for the parameter containers in libR3BAsyChimera           -----
+//         Created 03/11/19  by J.L. Rodriguez-Sanchez              -----
+//								    -----
+// ----------------------------------------------------------------------
+
```

Comment:
Please, update this class according to:
https://github.com/R3BRootGroup/R3BRoot/blob/dev/twim/pars/R3BTwimContFact.cxx
https://github.com/R3BRootGroup/R3BRoot/blob/dev/twim/pars/R3BTwimContFact.h